### PR TITLE
Upgrade relay

### PR DESCRIPTION
**Pull request by tbergq**
Closes: https://github.com/adeira/universe/issues/421

### CODE REVIEW

File: src/relay/src/hooks/useRelayEnvironment.js [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
 import type { Environment } from '../runtimeTypes.flow';
 
 export default function useRelayEnvironment(): Environment {
-  return _useRelayEnvironment();
+  const context = React.useContext(ReactRelayContext);
+  invariant(
+    context != null,
+    'Expected to be used inside a relay context, but was not. Make sure it is wrapped in a RelayEnvironmentProvider or QueryRenderer',
+  );
+  return context.environment;
```
Comment (mrtnzlml) [inline src/relay/src/hooks/useRelayEnvironment.js]: This is what we used before (basically just a better error message):

```js
// @flow

import { useContext } from 'react';
import { ReactRelayContext } from 'react-relay';
import { invariant } from '@adeira/js';

import type { Environment } from '../runtimeTypes.flow';

/**
 * You can use this hook in any component below QueryRenderer or RelayEnvironmentProvider.
 */
export default function useRelayEnvironment(): Environment {
  const context = useContext(ReactRelayContext);
  invariant(
    context != null,
    'useRelayEnvironment: Expected to have found a Relay environment provided by' +
      'a `RelayEnvironmentProvider` component or `QueryRenderer`. This usually means that' +
      'useRelayEnvironment was used in a component that is not a descendant of these components. ' +
      'Please make sure a `RelayEnvironmentProvider` has been rendered somewhere as a parent of ' +
      'your component. It should be defined only once in the whole application.',
  );
  return context.environment;
}
```

Comment (tbergq) [reply src/relay/src/hooks/useRelayEnvironment.js]: Sure, I will use the one from before

**Comment by tbergq** [comment]
> No changes in generated files? Did you try to run Relay Compiler on our example? :)

No changes, I ran it both in example and in relay

File: src/relay/src/hooks/useRelayEnvironment.js (hunk 1 above)
Comment (tbergq) [reply src/relay/src/hooks/useRelayEnvironment.js]: @mrtnzlml please recheck with new error message 🚀

**Pull request merged**